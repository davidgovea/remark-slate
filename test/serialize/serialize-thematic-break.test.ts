import { serialize, defaultNodeTypes } from '../../src';

it('Serialize a thematic break from slate state to markdown', () => {
  expect(
    serialize({
      type: defaultNodeTypes.thematic_break,
      children: [{ text: '' }],
    })
  ).toMatchSnapshot();
});
