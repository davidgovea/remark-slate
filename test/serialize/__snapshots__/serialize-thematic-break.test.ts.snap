// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Serialize a heading one from slate state to markdown 1`] = `
"---
"
`;
